parsers/US_SPP.py

```diff
@@ -66,14 +84,14 @@ def data_processor(df, logger):
 
     keys_to_remove = keys_to_remove | unknown_keys
 
+
     processed_data = []
     for index, row in df.iterrows():
         production = row.to_dict()
 
-        extra_unknowns = sum([production[k] for k in unknown_keys])
-        production['unknown'] = production['Other'] + production['Waste Heat'] + extra_unknowns
+        production['unknown'] = sum([production[k] for k in unknown_keys])
 
-        dt_aware = parser.parse(production['GMT MKT Interval'])
+        dt_aware = production['GMT MKT Interval']
```

Comment:
are we sure that the datetime returned in the file object is of type `datetime`?

Reply:
Yeah dateutil's `parser.parse()` will return a `datetime` object here.

Reply:
Ah yes, for the historical data I parsed that column earlier, so it is a timestamp, but I did not do that for the live data. I'll change that.

On a related note, for the downstream code, does the format have to be of type datetime.datetime? Or can it be a pandas timestamp? Because that may cause problems.

Reply:
Unless pandas timestamp can be cast to datetime.datetime I think it needs to be datetime.datetime. The reason is that it gets inserted into the database.

Reply:
This could be problematic in general since pandas wants all timestamps to be pandas timestamps, and it is more efficient to use parsed timestamps for filtering data. This is something we might need to think about finding a robust solution to in the future (unless there is one I am not seeing). In the meantime I will make it all datetimes without hopefully being too messy. :)

Reply:
sounds good -  I suggest you add this to our internal issue tracker?